clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -787,7 +787,8 @@ private KafkaConsumer(ConsumerConfig config, Deserializer<K> keyDeserializer, De
                         this.time,
                         enableAutoCommit,
                         config.getInt(ConsumerConfig.AUTO_COMMIT_INTERVAL_MS_CONFIG),
-                        this.interceptors);
+                        this.interceptors,
+                        config.getBoolean("throw.on.stable.flag.unsupported"));
```

Comment:
Why not use `THROW_ON_STABLE_FLAG_UNSUPPORTED` instead of hard-coded string

Reply:
Good idea